### DISCUSSION TITLE
IP-Adapter Composition Support

### DISCUSSION THREAD

**Discussion by enternalsaga** [Mikubill/sd-webui-controlnet#2774] [Ideas]
According to [this](https://www.reddit.com/r/StableDiffusion/comments/1bn2bsp/2_ipadapter_evolutions_that_help_unlock_more/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button) Any plans ahead to support it? I've tried Composition IPA, it works pretty well with my interior scene without destroying context, Thank you.

**Comment by CCpt5** [comment]
Have you seen: https://github.com/Mikubill/sd-webui-controlnet/discussions/2770 

Sorry if that's not what you're referring to.

**Comment by huchenlei** [comment]
Done in https://github.com/Mikubill/sd-webui-controlnet/pull/2779
